django/db/models/query.py

```diff
@@ -1792,6 +1792,9 @@ def _insert(
         Insert a new record for the given model. This provides an interface to
         the InsertQuery class and is how Model.save() is implemented.
         """
+        if self.model._meta.computed_fields and fields:
+            fields = [f for f in fields if f not in self.model._meta.computed_fields]
```

Comment:
When performing a `save()`, `create()` or `bulk_create()`, values for compute fields should not be specified

Reply:
Using `in <list>` leads to O(n²) runtime, can you not use the flag on the field?

```suggestion
        if self.model._meta.computed_fields and fields:
            fields = [f for f in fields if not f.computed]
```